src/v/config/configuration.cc

```diff
@@ -282,7 +282,7 @@ configuration::configuration()
        .example = "41943040",
        .visibility = visibility::tunable},
       std::nullopt,
-      {.min = 32_MiB})
+      {.min = 1_MiB})
```

Comment:
did you mean to change `raft_max_recovery_memory` min?  Should someone from replication verify this is ok?

Reply:
Nice catch this is definitely a git/texteditor error, meant to change an audit config

Reply:
This change is still present, did you mean to revert the change to `raft_max_recovery_memory`?

Reply:
Resolving now , latest force push addressed , nice catch